src/docs/patterns/icons.mdx

```diff
@@ -0,0 +1,172 @@
+-   Each uses their own images
+
+There are also a few icons commonly seen among Brightlayer UI applications, but less frequently seen elsewhere.
+
+-   Show a few other less obvious ones but still commonly used at Eaton
+    -   Logs, warning, error, star (favorite locations), device (assets), all apps 9 dots thing for software suite, toggle visibility of something (pwd), locations
+    -   A list, click to expand?
+    -   Maybe an option to "edit list on GitHub" for this list
+
+One should never use a universal icon in a distorted way that mismatches its original meaning.
+
+-   [Image 1: a confusing use of hamburger and waterfall icon.]
+
+-   [Image 2: a confusing use of settings and “service” / “repair”]
+
+## Common device icons
+
+Brightlayer UI offers a wide variety of icons for industrial equipment-related icons. Some of them are designed based on their function, while others are designed based on how the device itself looks. We also offer symbols commonly used in one-line diagrams (link to the two NPM packages; Do we want to call those out?).
+
+While we try our best to add new icons as soon as we identify a need, we noticed that many concepts are very closely related to each other, and it is often unwise to use icons for every single parameter in every single device, while expecting users to distinguish them immediately.
+
+-   [Image 1: AVOID: a list of devices, with a new icon for every single device]
+
+-   [Image 2: use the same icon for the same type of device]
+    Icon Variations
+    Multi-Colored Icons
+
+WCAG 2.2 AA level requires that non-textual contents of the webpage must achieve a minimum color contrast of 3:1. However, yellow icons are used for warnings and often have a contrast ratio of less than 2:1, therefore failing this requirement. A secondary color would make this icon more legible.
+
+-   [Image: show a warning icon, with a background in yellow and foreground in brown]
+
+## Filled and Outlined Icons
+
+Brightlayer UI applications may choose from two styles of icons: filled and outlined. You must ensure your icon style stays consistent through the application
+
+-   [Show an example of a dashboard list. All icons in a list should be either all filled or all outlined.]
+
+You also have the option to use filled and outlined icons to indicate state. In this case, a filled icon indicates that the item is active, while an outlined icon indicates that the item is inactive.
+
+-   [Show an example of a hero banner, with different statuses. If the count is 0, make it inactive; if it is a positive number, make it active]
+
+## Progress Icons
+
+Progress icons are used as a mini data visualization to display percentages over a whole. It can be the remaining device's health, battery life percentage, etc.
+
+-   [Show examples of percentage icons used in a real application]
+
+Brightlayer currently offers the following progress icons to its designers and developers:
+
+<ProgressIconCard />
```

Comment:
Do we want to remove this from the Iconography page?

Reply:
This should get removed in a separate PR. Right now this icons guide page is hidden, because I don't know where to put it yet. Once this icon guide page goes live, we can take the ProgressIconCard out from the other side.